TU20Bot/Handler.cs

```diff
@@ -62,7 +68,7 @@ public class Handler {
         public async Task userJoined(SocketGuildUser user) {
             var channel = (IMessageChannel) client.GetChannel(737081385583378447);
```

Comment:
I would recommend placing the channel ID into a readonly variable so that we can easily identify what it is accessing.